test/test_query_result.rb

```diff
@@ -243,7 +243,7 @@ def test_define_simple_query_result
     GRAPHQL
 
     response = @client.query(Temp::Query)
-    refute response.data.me.respond_to?(:name)
+    assert response.data.me.respond_to?(:name)
```

Comment:
this feels wrong, I'll try to understand why this behaved differently when method is explicitly defined